game/src/Credits/Credits.gd

```diff
@@ -32,72 +32,85 @@ var label_settings_personel : LabelSettings
 @export
 var credits_list: VBoxContainer
 
+const title_str : String = "TITLE"
```

Comment:
Adding str doesn't really explain what its for

```suggestion
const title_key: String = "TITLE"
```

Reply:
I agree `title_key` is a better name. On whether to have a space before the colon in typed arguments, the only file in the repository currently using the "no space" style is `Options.gd`, so I'll leave that to a wider formatting PR.

Reply:
Yeah sure, no space thing doesn't matter that much to me just thought I'd mention it